providers/common/der/build.info

```diff
@@ -97,9 +97,6 @@ ENDIF
 
 #----- Conclusion
 
-# TODO(3.0) $COMMON should go to libcommon.a, but this currently leads
```

Comment:
Looks like this TODO was here for a reason :)

Reply:
Except it is not a problem of linking conflicts.

Reply:
So what is the problem according to you?

Reply:
keep reading..

Reply:
I would say, though, that *none* of the stuff here should have to go to libfips.a or libdefault.a.  It should really all go to libcommon.a...

...  and now that I look more closely, I see no reason why it's not.  I'll go digging a bit.

Reply:
or well, I can see reason why some of it would only end up in libdefault.a

Reply:
See #15338.  With *that*, this TODO is fixed in my view